pokecli.py

```diff
@@ -185,7 +186,7 @@ def get_commit_hash():
         finished = False
 
         while not finished:
-            wait_time = config.reconnecting_timeout * 60
+            wait_time = randint(config.reconnecting_timeout, (config.reconnecting_timeout * 60))
```

Comment:
I think this needs adjustment and docs changes: previously, config.reconnecting_timeout was "time in minutes". Now the wait_time spans from this valud to the 60-times of it.

Maybe `randint(config.reconnecting_timeout*60, (config.reconnecting_timeout + 1) * 60))` ?